admin_guide/scheduling/node_affinity.adoc

```diff
@@ -18,7 +18,7 @@ For example, you could configure a pod to only run on a node with a specific CPU
 
 There are two types of node affinity rules: _required_ and _preferred_.
 
-Required rules *must* be met before a pod can be scheduled on a node. Preferred rules specify that, if the rule is met, the scheduler will try to enforce the rules, but does not guarantee enforcement.
+Required rules *must* be met before a pod can be scheduled on a node. Preferred rules specify that, if the rule is met, the scheduler will try to enforce the rules, but does not guarantee enforcement. If a pod has an affinity rule for which there is no node with a matching label, that pod can be scheduled on a node that does not have a label 
```

Comment:
@wmengRH 
"If a pod has an affinity rule, but there is no node with a matching label, that pod can be scheduled on a node that does not have a label selector. "